app/views/draws/show.html.erb

```diff
@@ -1,3 +1,19 @@
 <h1 class="draw-name"><%= @draw.name %></h1>
+<div class="intent-metrics">
```

Comment:
should this be in a partial?

also, I *think* `#show` is visible to students as well as admins, so the permissions statement should probably include the metric report